api/index.md

```diff
@@ -5,13 +5,570 @@ _navPath: toc.html
+   <td>
+   Provides cryptographic services, including secure encoding and decoding of data, as well as many other operations, such as hashing, random number generation, and message authentication. 
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.Security.Cryptography.Pkcs.html">System.Security.Cryptography.Pkcs</a></td>
+   <td>
+   Provides programming elements for Public Key Cryptography Standards (PKCS), including methods for signing data, exchanging keys, requesting certificates, public key encryption and decryption, and other security functions.
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.Security.Cryptography.X509Certificates.html">System.Security.Cryptography.X509Certificates</a></td>
+   <td>
+   Contains the runtime's implementation of the Authenticode X.509 v.3 certificate. This certificate is signed with a private key that uniquely and positively identifies the holder of the certificate.
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.Security.Cryptography.Xml.html">System.Security.Cryptography.Xml</a></td>
+   <td>
+   Contains a single type that represents the **<X509IssuerSerial>** element of an XML digital signature.
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.Security.Principal.html">System.Security.Principal</a></td>
+   <td>
+   Defines a principal object that represents the security context under which code is running.  
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.ServiceProcess.html">System.ServiceProcess</a></td>
+   <td>
+   Provides types that allow you to connect to and control a Windows service application. Windows services are long-running executables with no user interface. 
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.Text.html">System.Text</a></td>
+   <td>
+   Contains classes that represent ASCII and Unicode character encodings; abstract base classes for converting blocks of characters to and from blocks of bytes; and a helper class that manipulates and formats `String` objects without creating intermediate `String` instances.
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.Text.RegularExpressions.html">System.Text.RegularExpressions</a></td>
+   <td>
+   Contains classes that represent the .NET regular expression engine, which allows you to search text for particular patterns. 
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.Threading.html">System.Threading</a></td>
+   <td>
+   Provides types that enable multithreaded programming. In addition to classes for synchronizing thread activities and access to data (`Mutex`, `Monitor`, `Interlocked`, `AutoResetEvent`, and so on), this namespace includes a `ThreadPool` class that allows you to use a pool of system-supplied threads, and a `Timer` class that executes callback methods on thread pool threads.
```

Comment:
in some cases you're linking to the class, in others formatting as code. I think it would be good to be consistent